scripts/local-dev.sh

```diff
@@ -22,39 +22,81 @@
 
 set -euo pipefail
 
+MINIKUBE_PROFILE_NAME="${MINIKUBE_PROFILE_NAME:-parca-agent}"
+function mk() {
+    minikube -p "${MINIKUBE_PROFILE_NAME}" "$@"
+}
+
 # Creates a local minikube cluster, and deploys the dev env into the cluster
 function up() {
-  # Spin up local cluster if one isn't running
-  if minikube status -p parca-agent; then
-    echo "----------------------------------------------------------"
-    echo "Dev cluster already running. Skipping minikube cluster creation"
-    echo "----------------------------------------------------------"
-  else
-    ctlptl create registry ctlptl-registry || echo 'Registry already exists'
-    # kvm2, hyperkit, hyperv, vmwarefusion, virtualbox, vmware, xhyve
-    minikube start -p parca-agent \
-      --driver=kvm2 \
-      --kubernetes-version=v1.22.3 \
-      --cpus=4 \
-      --memory=16gb \
-      --disk-size=20gb \
-      --docker-opt dns=8.8.8.8 \
-      --docker-opt default-ulimit=memlock=9223372036854775807:9223372036854775807
-  fi
-
-  # Deploy all services into the cluster
-  deploy
-
-  echo "Now run \"tilt up\" to start developing!"
+    # Spin up local cluster if one isn't running
+    if mk status; then
+        echo "----------------------------------------------------------"
+        echo "Dev cluster already running. Skipping minikube cluster creation"
+        echo "----------------------------------------------------------"
+    else
+        # local_registry
+
+        echo "----------------------------------------------------------"
+        echo "Creating minikube cluster"
+        echo "----------------------------------------------------------"
+        # kvm2, hyperkit, hyperv, vmwarefu1sion, virtualbox, vmware, xhyve
+        mk start \
+            --driver=kvm2 \
+            --nodes=3 \
+            --kubernetes-version=v1.23.3 \
+            --cpus=2 \
+            --memory=8gb \
```

Comment:
More reasonable defaults :+1:, my local machine literally has 4CPU/16GB only :sweat_smile:

Reply:
:rofl: Yes, this is one of the things I'll clean up before merging. Thanks for pointing out.